server/plugin.go

```diff
@@ -41,7 +42,8 @@ const (
 
 type Plugin struct {
 	plugin.MattermostPlugin
-	githubClient *github.Client
+	githubClient         *github.Client
```

Comment:
should this be removed?

Reply:
Yeah, it can. But now it's unrelated to the PR. This field was never used. I can do it if you want.

Reply:
Since it's not being used I'd recommend to remove it. No harm if it's not being used.